src/askem_beaker/contexts/mira_model_edit/agent.py

```diff
@@ -644,7 +644,8 @@ async def stratify(self,
         concepts_to_stratify: Optional[list[str]] = None,
         concepts_to_preserve: Optional[list[str]] = None,
         params_to_stratify: Optional[list[str]] = None,
-        params_to_preserve: Optional[list[str]] = None
+        params_to_preserve: Optional[list[str]] = None,
+		add_param_factor = Optional[bool] = True
```

Comment:
Careful to not mix tabs and spaces.

Reply:
Yep thanks, this hasn't been tested yet.